hw2/04-background.js

```diff
@@ -1 +1,50 @@
 // Add your code here
+const btn = document.getElementById('control');
+let intervalTime = document.getElementById('intervalTime');
+let nIntervId;
+//function from:
+//https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min) + min);
+}
+
+function changeColor() {
+  const background = document.querySelector('body');
+  const red = getRandomInt(0, 255);
+  const blue = getRandomInt(0, 255);
+  const green = getRandomInt(0, 255);
+  background.style.backgroundColor = `rgba(${red}, ${green}, ${blue}, 0.5)`;
+}
+
+function startIntervals(sec) {
+  const time = sec * 1000;
+  if (!nIntervId) {
+    nIntervId = setInterval(changeColor, time);
+  }
+  btn.textContent = 'Stop';
+  btn.style.backgroundColor = 'red';
```

Comment:
maybe add bootstrap classes to the buttons rather than adjusting the style directly